sqrl-testing/sqrl-integration-tests/src/test/resources/usecases/avro-schema/avro-schema/schema.avsc

```diff
@@ -0,0 +1,370 @@
+    },
+    {
+      "name": "timestampMicrosField",
+      "type": {
+        "type": "long",
+        "logicalType": "timestamp-micros"
+      },
+      "doc": "A timestamp with microsecond precision"
+    },
+    {
+      "name": "nullableTimestampMicrosField",
+      "type": ["null", {
+        "type": "long",
+        "logicalType": "timestamp-micros"
+      }],
+      "default": null,
+      "doc": "A nullable timestamp with microsecond precision"
+    },
+//    {
+//      "name": "dateField",
+//      "type": {
+//        "type": "int",
+//        "logicalType": "date"
+//      },
+//      "doc": "A date field representing days since the UNIX epoch"
+//    },
+//    {
+//      "name": "nullableDateField",
+//      "type": ["null", {
+//        "type": "int",
+//        "logicalType": "date"
+//      }],
+//      "default": null,
+//      "doc": "A nullable date field"
+//    },
+//    {
+//      "name": "timeMillisField",
+//      "type": {
+//        "type": "int",
+//        "logicalType": "time-millis"
+//      },
+//      "doc": "A time field with millisecond precision"
+//    },
+//    {
+//      "name": "nullableTimeMillisField",
+//      "type": ["null", {
+//        "type": "int",
+//        "logicalType": "time-millis"
+//      }],
+//      "default": null,
+//      "doc": "A nullable time field with millisecond precision"
+//    },
+//    {
+//      "name": "timeMicrosField",
+//      "type": {
+//        "type": "long",
+//        "logicalType": "time-micros"
+//      },
+//      "doc": "A time field with microsecond precision"
+//    },
+//    {
+//      "name": "nullableTimeMicrosField",
+//      "type": ["null", {
+//        "type": "long",
+//        "logicalType": "time-micros"
+//      }],
+//      "default": null,
+//      "doc": "A nullable time field with microsecond precision"
+//    },
+//    {
+//      "name": "complexArrayField",
+//      "type": {
+//        "type": "array",
+//        "items": {
+//          "type": "record",
+//          "name": "ArrayItemRecord",
+//          "fields": [
+//            {
+//              "name": "itemFieldOne",
+//              "type": "int"
+//            },
+//            {
+//              "name": "itemFieldTwo",
+//              "type": "string"
+//            }
+//          ]
+//        }
+//      },
+//      "doc": "An array of complex nested records"
+//    },
+//    {
+//      "name": "nullableComplexArrayField",
+//      "type": ["null", {
+//        "type": "array",
+//        "items": {
+//          "type": "record",
+//          "name": "NullableArrayItemRecord",
+//          "fields": [
+//            {
+//              "name": "nullableItemFieldOne",
+//              "type": ["null", "double"],
+//              "default": null
+//            },
+//            {
+//              "name": "nullableItemFieldTwo",
+//              "type": ["null", "boolean"],
+//              "default": null
+//            }
+//          ]
+//        }
+//      }],
```

Comment:
Automatically generated test queries do not include nested subselection and will produce invalid queries